### PR TITLE
chore(fxa-client): removingparam "style" from /sessions/verify_code from both the frontend and backend 

### PR DESCRIPTION
**Pull request by Cronus1007**
## Because
This param is a left over from the trailhead experiment. It should be safe to remove in frontend and  backend.
 -

## This pull request

-

## Issue that this pull request solves

Closes: #6058 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

**Comment by Cronus1007** [comment]
cc @meandavejustice @dannycoates  @vbudhram

**Comment by Cronus1007** [comment]
cc @chenba

**Comment by chenba** [comment]
@Cronus1007 Thanks for another PR!

- Feel free to delete code (vs. commenting out).
- I don't see front end changes.  Has `style` been removed from the request payload there?
- If you use the "Reviewers" feature to request a review from `fxa-devs`, then the team will be notified.

**Comment by Cronus1007** [comment]
@chenba Everything works fine on my end

**Comment by Cronus1007** [comment]
@chenba 

- Ok I will take care

- Everything seems to work fine on my end. I think that you are having a look at the deployed version of main branch. I have commited changes on removingparam branch(my branch) . This time CircleCi build has been through removingparam branch itself 

- Ok Surely I will use it.

File: packages/fxa-auth-server/lib/routes/session.js [hunk 1]
```diff
@@ -329,7 +329,7 @@ module.exports = function (
           payload: {
             code: validators.DIGITS,
             service: validators.service,
-            style: validators.style,
+            // Removed validators.style from the payload
```
Comment (chenba) [inline packages/fxa-auth-server/lib/routes/session.js:332]: I don't think we need this comment.

**Comment by chenba** [comment]
@Cronus1007 Looks like there's another spot where `style` still exist: https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-client/lib/client.ts#L561

File: packages/fxa-auth-server/lib/routes/session.js (hunk 1 above)
Comment (Cronus1007) [reply packages/fxa-auth-server/lib/routes/session.js:332]: Okk

**Comment by Cronus1007** [comment]
@chenba I have made the desired changes.

**Comment by chenba** [comment]
@Cronus1007 Thanks for making those changes.  Just a couple more process related things before merging.

1. Please squash all the commits into a single commit.
1. Please update the commit message to conform with https://github.com/mozilla/fxa/blob/main/CONTRIBUTING.md#git-commit-guidelines; this is a "chore".
1. Please sign your commit.  (See https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/managing-commit-signature-verification for a guide.)

Thanks!

**Comment by Cronus1007** [comment]
@chenba  Done.

**Comment by chenba** [comment]
@Cronus1007 I still see four unsigned commits: https://github.com/mozilla/fxa/pull/6850/commits

**Comment by Cronus1007** [comment]
@chenba  I have merged all the commits into https://github.com/mozilla/fxa/pull/6850/commits/3f2f249e6d20e535b8c25231d691bb3b4f5bc6e6
commit.

**Comment by chenba** [comment]
@Cronus1007 The PR now has six commits.  Here's a guide on how to use `git rebase -i` to squash commits: https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

**Comment by Cronus1007** [comment]
@chenba  I think that I should close this PR and start with a new fork and commit all the changes in a single commit.

**Pull request closed without merging**